app/services/id_token_builder.rb

```diff
@@ -65,6 +75,18 @@ def acr
     end
   end
 
+  def aal_acr
+    aal = identity.aal || ::Idp::Constants::DEFAULT_AAL
+    case aal
+    when Idp::Constants::DEFAULT_AAL then Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF
+    when Idp::Constants::AAL1 then Saml::Idp::Constants::AAL1_AUTHN_CONTEXT_CLASSREF
+    when Idp::Constants::AAL2 then Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF
+    when Idp::Constants::AAL3 then Saml::Idp::Constants::AAL3_AUTHN_CONTEXT_CLASSREF
```

Comment:
personally I'd rather have a newline than the `then` keyword, but I'm not sure if we have a codebase convention/rubocop rule around that
```suggestion
    when Idp::Constants::DEFAULT_AAL
      Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF
    when Idp::Constants::AAL1
      Saml::Idp::Constants::AAL1_AUTHN_CONTEXT_CLASSREF
    when Idp::Constants::AAL2
      Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF
    when Idp::Constants::AAL3
      Saml::Idp::Constants::AAL3_AUTHN_CONTEXT_CLASSREF
```